academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -1354,13 +83,13 @@ If you have not done so already, download the kesm.exe tool from your MAKES subs
 
     | Values | Description | Example |
```

Comment:
>Values [](start = 6, length = 6)

In the private build of "kesm.exe" usage (when the user just type kesm.exe and then press enter), there is no “—AzureSubscriptionId” param.